tests/series_only/is_between_test.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pytest
+
+import narwhals.stable.v1 as nw
+from tests.utils import compare_dicts
+
+data = [1, 4, 2, 5]
+
+
+@pytest.mark.parametrize(
+    ("closed", "expected"),
+    [
+        ("left", [True, True, True, False]),
+        ("right", [False, True, True, True]),
+        ("both", [True, True, True, True]),
+        ("none", [False, True, True, False]),
+    ],
+)
+def test_is_between(
+    request: Any, constructor_series: Any, closed: str, expected: list[bool]
+) -> None:
+    if "pandas_series_nullable_constructor" in str(constructor_series):
+        request.applymarker(pytest.mark.xfail)
+
```

Comment:
sorry I meant - you can just remove this completely

Reply:
aaaah okay, sorry for the misunderstanding. Really curious why more checks are failing without it now.

Reply:
because now you're telling pytest "check that this fails for pandas nullable series" - however, the implementation passes for all libraries and all versions, so we don't need any `pytest.mark.xfail` 😎

Reply:
Now I see 🙈, I didn't notice that you marked the entire if stetement, but just removed it and noticed it worked. Thank you why though!